### PR TITLE
Update zh_cn localization

### PR DESCRIPTION
**Pull request by RawPineapple**
- Refined overall translation
- Changed formatting so they look neat in game GUI
- Added missing translations

problem: some block/Entity/item names are still displayed in English due to being passed by identifiers. Nothing I can do on the language file side.

### CODE REVIEW

File: src/main/resources/assets/levelz/lang/zh_cn.json [hunk 1]
```diff
@@ -27,194 +27,194 @@
...
-  "text.autoconfig.levelz.option.miningLockedMultiplicator": "已锁定的方块破坏速度乘数",
+  "text.autoconfig.levelz.option.miningOreChance": "额外矿物掉落几率",
+  "text.autoconfig.levelz.option.miningLockedMultiplicator": "已锁定的方块破坏速度减慢倍数",
   "text.autoconfig.levelz.option.highlightLocked": "高亮显示锁定的方块",
   "text.autoconfig.levelz.option.disableFurnaceXP": "关闭熔炉经验",
-  "text.autoconfig.levelz.option.dropPlayerXP": "Drop Player XP",
+  "text.autoconfig.levelz.option.dropPlayerXP": "玩家经验掉落",
 
-  "item.levelz.strange_potion": "Strange Potion",
+  "item.levelz.strange_potion": "奇怪药水",
```
Comment (Moirstral) [inline src/main/resources/assets/levelz/lang/zh_cn.json]: 神奇药水会好些

File: src/main/resources/assets/levelz/lang/zh_cn.json [hunk 2]
```diff
@@ -1,14 +1,14 @@
...
   "category.levelz.keybind": "LevelZ 按键绑定",
-  "key.levelz.openskillscreen": "打开技能点GUI",
+  "key.levelz.openskillscreen": "打开技能界面",
 
   "text.autoconfig.levelz.title": "LevelZ 配置",
   "text.autoconfig.levelz.category.level_setting": "等级设置",
   "text.autoconfig.levelz.category.default": "技能设置",
   "text.autoconfig.levelz.option.hardMode": "极限模式 (开启后玩家死亡时清除所有点数和经验)",
-  "text.autoconfig.levelz.option.disableMobFarms": "Disable Mob Farms",
+  "text.autoconfig.levelz.option.disableMobFarms": "经由刷怪笼生成的怪物不给予经验值",
```
Comment (Moirstral) [inline src/main/resources/assets/levelz/lang/zh_cn.json]: 这块我看了代码，并不是刷怪笼，而是一个区块里杀怪超过 5 只，就会不掉任何东西，当然刷怪笼会触发更频繁，所以你可能是误解了。直接翻译为 “禁用刷怪塔掉落” 即可。

**Review by Moirstral — CHANGES_REQUESTED** [review]
翻译有部分错误，建议修改后再 PR

File: src/main/resources/assets/levelz/lang/zh_cn.json [hunk 3]
```diff
@@ -27,194 +27,194 @@
   "text.autoconfig.levelz.option.defenseBase": "基础防御",
   "text.autoconfig.levelz.option.defenseBonus": "每级防御加成",
   "text.autoconfig.levelz.option.defenseReflectChance": "伤害反弹几率",
-  "text.autoconfig.levelz.option.luckBase": "基础幸运",
-  "text.autoconfig.levelz.option.luckBonus": "每级幸运加成",
+  "text.autoconfig.levelz.option.luckBase": "基础幸运值",
+  "text.autoconfig.levelz.option.luckBonus": "每级幸运值加成",
   "text.autoconfig.levelz.option.luckCritBonus": "每级幸运一击加成",
-  "text.autoconfig.levelz.option.luckSurviveChance": "免死几率",
+  "text.autoconfig.levelz.option.luckSurviveChance": "幸存致死伤害几率",
```
Comment (Moirstral) [inline src/main/resources/assets/levelz/lang/zh_cn.json]: 语法并不通顺，建议翻译为 “免疫致死伤害几率”

**Comment by RawPineapple** [comment]
Fixed incorrect translations
Taking suggestions by @LangYueMc

**Comment by RawPineapple** [comment]
tested in game, not seeing any translation error. You may merge this safely now.

**Comment by Globox1997** [comment]
Thanks for contributing :)

**Pull request merged**